doc/source/merging.rst

```diff
@@ -86,12 +86,18 @@ some configurable handling of "what to do with the other axes":
   it is passed, in which case the values will be selected (see below). Any None
   objects will be dropped silently unless they are all None in which case a
   ValueError will be raised.
-- ``axis`` : {0, 1, ...}, default 0. The axis to concatenate along.
+- ``axis`` : {0, 1, ...}, default 0. The axis to concatenate along. 0 is the index
```

Comment:
make this more similar to the doc-string of ``pd.concat`` (you can change either), but it should follow our existing conventions, and this is not it

Reply:
I am not sure what you ask me to do when you say "make more similar". The styles slightly differ and I tried to follow the styles I found.

Reply:
reread my comment 

similar to the doc string as i said